core-web/src/main/java/org/silverpeas/core/webapi/profile/UserProfileService.java

```diff
@@ -64,4 +66,14 @@ public Group getGroupAccessibleToUser(String groupId, final UserDetail user) thr
     }
     return theGroup;
   }
+
+  public List<Group> getGroupsAccessibleToUser(List<String> groupIds, final UserDetail user)
+      throws WebApplicationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'javax.ws.rs.WebApplicationException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)